rbac/management/migrations/0011_group_naming.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.4 on 2019-11-26 17:03
+
+from django.db import migrations, models
+
+
+def update_default_name(apps, schema_editor):
+    Group = apps.get_model('management', 'Group')
+    for group in Group.objects.all():
+        if group.system:
+            group.name = "Default user access"
```

Comment:
Just ran this locally, and my existing system groups didn't update. I think we want to do `group.save()` after this?
```suggestion
            group.name = "Default user access"
            group.save()
```